skimage/transform/tests/test_thin_plate_splines_transform.py

```diff
@@ -119,27 +136,26 @@ def test_tps_transform_call():
     # Test __call__ method with estimmate
     tform.estimate(SRC, DST)
 
-    transformed_pts = tform(coords)
-    xx = transformed_pts[0]
-    yy = transformed_pts[1]
+    xx_trans, yy_trans = tform(coords)
 
-    assert len(transformed_pts) == 2
-    assert xx.shape == (5, 5)
-    assert yy.shape == (5, 5)
+    assert xx_trans.shape == (5, 5)
+    assert yy_trans.shape == (5, 5)
 
-    expected_xx = np.array([[5., 5., 5., 5., 5.],
-                            [4., 4., 4., 4., 4.],
-                            [3., 3., 3., 3., 3.],
-                            [2., 2., 2., 2., 2.],
-                            [1., 1., 1., 1., 1.]])
-    np.testing.assert_allclose(xx, expected_xx)
+    expected_yy = np.array([[0, 1.0, 2.0, 3.0, 4.0],
+                            [0, 1.0, 2.0, 3.0, 4.0],
+                            [0, 1.0, 2.0, 3.0, 4.0],
+                            [0, 1.0, 2.0, 3.0, 4.0],
+                            [0, 1.0, 2.0, 3.0, 4.0]], dtype=np.float32)
```

Comment:
I find it interesting that the y-dimension has the range [0, 4], while the x-dimension is [1, 5]. Not a blocker, but it's really annoying me that I can't figure out the reason behind this.

Reply:
Could it have something to do with the part of paper that states "the x-coordinate is transferred from left to right without change, while the y-coordinate is altered by the value `z(x,y)`... " ? You can find this 568, section C.